.github/workflows/query-engine-driver-adapters.yml

```diff
@@ -28,20 +28,24 @@ jobs:
             version: "13"
             driver_adapter: "pg"
         node_version: ["18"]
-
     env:
       LOG_LEVEL: "info"
       LOG_QUERIES: "y"
       RUST_LOG: "info"
       RUST_LOG_FORMAT: "devel"
       RUST_BACKTRACE: "1"
+      PRISMA_DISABLE_QUAINT_EXECUTORS: "1"
       CLICOLOR_FORCE: "1"
       CLOSED_TX_CLEANUP: "2"
       SIMPLE_TEST_MODE: "1"
       QUERY_BATCH_SIZE: "10"
       TEST_CONNECTOR: ${{ matrix.database.connector }}
       TEST_CONNECTOR_VERSION: ${{ matrix.database.version }}
-      TEST_DRIVER_ADAPTER: ${{ matrix.database.driver_adapter }}
+      WORKSPACE_ROOT: ${{ github.workspace }}
+      # Driver adapter testing specific env vars
+      NODE_TEST_EXECUTOR: "${{ github.workspace }}/query-engine/driver-adapters/js/connector-test-kit-executor/script/start_node.sh"
+      DRIVER_ADAPTER: ${{ matrix.database.driver_adapter }}
+      DRIVER_ADAPTER_URL_OVERRIDE: ${{ matrix.database.driver_adapter_url }}
```

Comment:
What will the value be here when `driver_adapter_url` is not specified for the item in matrix (like for pg)?

Reply:
it would be the empty string, but given the (fixed) code in:

```js
const connectionString = process.env.DRIVER_ADAPTER_URL_OVERRIDE ?? ''
if (connectionString == '') {
    throw new Error("DRIVER_ADAPTER_URL_OVERRIDE is not defined or empty, but its required for neon adapter.");
}
```

It will work as expected.